Formula/tg.rb

```diff
@@ -0,0 +1,28 @@
+class Tg < Formula
+  include Language::Python::Virtualenv
+
+  desc "Terminal telegram client"
+  homepage "https://github.com/paul-nameless/tg"
+  url "https://github.com/paul-nameless/tg/archive/refs/tags/v0.11.0.tar.gz"
+  sha256 "2303983af7b7246fb3198bcae688d3b6327a96bbfc91c94ef9bc783cc7191f42"
+  license "Unlicense"
+
+  bottle :unneeded
+
+  depends_on "python@3.9"
+  depends_on "tdlib" => "1.7.0"
```

Comment:
You can't specify versions here:
```suggestion
  depends_on "tdlib"
```